framework/qlf/dashboard/bokeh/qacountbins/main.py

```diff
@@ -62,6 +66,7 @@
 
 
 countbins = ast.literal_eval(countbins)
+params_countbins = ast.literal_eval(tests['countbins'])
```

Comment:
We changed the database so there is no need to use `ast.literal_eval` anymore anywhere.